static/src/javascripts/projects/common/modules/identity/api.js

```diff
@@ -24,6 +25,11 @@ type PasswordCredential = {
     password: string,
 };
 
+export type SettableConsent = {
+    id: string,
```

Comment:
you can reuse the Consent type we already have, better still `Followable<Consent>[]`

Reply:
okay!